File: Assets/Objects/Objects.js
#pragma strict
var System : GameObject;
var Taken : boolean = false;
var Type : String = "SitPlace";

function Start () 
{
	if(Type == "SitPlace" || Type == "WorkPlace" )
	{
		System.GetComponent(ObjectLoop).Bench.Add(this.gameObject);
	}
	
	if(Type == "PlayPlace")
	{
		System.GetComponent(ObjectLoop).PlayPlace.Add(this.gameObject);

	}
}

function Update () 
{
	
}

File: Assets/Objects/Objects.cs
using UnityEngine;

public class Objects : MonoBehaviour
{
    public GameObject System;
    public bool Taken = false;
    public string Type = "SitPlace";

    // Use this for initialization
    void Start()
    {
        if (Type == "SitPlace" || Type == "WorkPlace")
        {
            System.GetComponent<ObjectLoop>().Bench.Add(this.gameObject);
        }

        if (Type == "PlayPlace")
        {
            System.GetComponent<ObjectLoop>().PlayPlace.Add(this.gameObject);
        }
    }

    // Update is called once per frame
    void Update()
    {

    }
}
